packages/react-icons/src/utils/createFluentIcon.ts

```diff
@@ -12,11 +13,22 @@ export type CreateFluentIconOptions = {
     color?: boolean;
 }
 
+const useRootStyles = makeStyles({
```

Comment:
Maybe one comment here too, about how this is because browsers automatically have SVGs set to `forced-color-adjust: preserve-parent-color`, which will not override internally-defined colors in our SVG icons in high contrast mode.